feature-libs/organization/administration/core/services/user-group.service.ts

```diff
@@ -69,25 +68,25 @@ export class UserGroupService {
   private getUserGroupValue(userGroupId: string): Observable<Budget> {
```

Comment:
It seems as though the return type of the method is wrong and should be `Observable<UserGroup>` instead of `Observable<Budget>`.  What do you think?

Reply:
yes, you are right!